src/gitlab.js

```diff
@@ -19,18 +19,52 @@ class GitLab
     return request(options);
   }
 
-  getProjects() {
+  getProject({ page = 1 }) {
```

Comment:
The name of this function would imply it only gets 1 project. I see this is an internal function now, what do you think about the name `_getProjects` ?

Reply:
Yes, I can change it.